mm2src/coins/Cargo.toml

```diff
@@ -23,11 +23,13 @@ doctest = false
 async-std = { version = "1.5", features = ["unstable"] }
 async-trait = "0.1.52"
 base64 = "0.21.2"
+# Todo: remove this and rely on bs58 throught the whole codebase
```

Comment:
Can we just get rid of one of `base58` and `bs58`? Both seems to be doing the same thing with `bs58` being faster if I'm correct.

Reply:
I guess you missed this https://github.com/KomodoPlatform/komodo-defi-framework/pull/2427

Reply:
P.S. I am trying to adhere to the principle of maintaining a single purpose or context per PR as per the review guidelines.